src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -151,6 +155,8 @@ export interface IChartContainerProps {
     selectedGenes?: any;
     cancerGenes: number[];
     onGeneSelect?: any;
+    selectedStructuralVariants?: any;
```

Comment:
seems like we could improve these types

Reply:
Do you have suggestions?

Reply:
I mean, these are quite specific types are they not?   just because 'any' got through before, lets not dig deeper.

Reply:
Fixed in [1cc6c5c](https://github.com/cBioPortal/cbioportal-frontend/pull/4577/commits/1cc6c5c31419dff689737a8c24242de7e719823d)